packages/addon-rslib/src/init.ts

```diff
@@ -0,0 +1,57 @@
+import { createRsbuild, mergeRsbuildConfig } from '@rsbuild/core'
+import { unstable_composeCreateRsbuildConfig } from '@rslib/core'
+
+import type { RsbuildConfig, RsbuildInstance } from '@rsbuild/core'
+import type { RslibConfig } from '@rslib/core'
+
+export async function startMFDevServer(
+  config: RslibConfig,
+): Promise<RsbuildInstance | undefined> {
+  if (process.env.NODE_ENV === 'production') {
+    // storybook build should skip this.
+    return
+  }
+  const rsbuildInstance = await initMFRsbuild(config)
+  if (!rsbuildInstance) {
+    // no mf format, return.
+    return
+  }
+  await rsbuildInstance.startDevServer()
```

Comment:
what if we move this line into `initMFRsbuild` to make the MF functionality more more cohesive.